github/ci/prow-deploy/hack/deploy.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+set -e
+
+main(){
+    current_dir=$(dirname "$0")
+    project_infra_root=$(readlink -f "${current_dir}/../../../..")
+
+    base_dir=${project_infra_root}/github/ci/prow-deploy
+
+    source ${project_infra_root}/hack/manage-secrets.sh
+    decrypt_secrets
```

Comment:
nit: maybe we could add the target directory where to put the file and target file name as an argument (in another PR of course)?

Reply:
Sure, sgtm :+1: